projects/map-three/src/lib/map-three.component.scss

```diff
@@ -0,0 +1,5 @@
+.threeCanvas {
+    width: 100%;
+    height: 600px;
```

Comment:
Always try to set sizes in procent not in pixel, otherwise the user has to overwrite it.

Reply:
Done!